.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: YgA8hpToZwO7tMdm7owflbzfogJ7Z4mzQ
```

Comment:
Why are you exposing the repo token?

Reply:
This should be hidden by using an environment variable. use this environment variable in Travis `CODECLIMATE_REPO_TOKEN`. and set the token as its value there

Reply:
Okay. I've resolve it.
Thanks.